<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>405</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>821</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>821</width>
    <height>405</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>-3</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; text-decoration: underline; color:#030070;&quot;&gt;Veranda Data Record&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>321</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>38</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log :</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>60</y>
      <width>330</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Hostname</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="host">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Database Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dbname">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="user">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pass">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>208</y>
      <width>333</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Start Time</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="startsUTC">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>End Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="endUTC">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Mobile ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="mobile_id">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>321</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>713</x>
      <y>36</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Screen</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="memo">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>60</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
